Simulator/Connections/Neuro/ConnGrowth.cpp

```diff
@@ -254,7 +254,7 @@ void ConnGrowth::updateSynapsesWeights()
          BGSIZE synapse_adjusted = 0;
          BGSIZE iEdg = Simulator::getInstance().getMaxEdgesPerVertex() * destVertex;
          for (BGSIZE synapseIndex = 0; synapse_adjusted < synapseCounts; synapseIndex++, iEdg++) {
-            if (synapses.inUse_[iEdg] == true) {
+            if (synapses.inUse_[iEdg] == 1) {
```

Comment:
I hate to be "that guy", but I can't help it. Is there a standard for conversion between `unsigned char` and the `const bool` values `true` and `false`? (Well, clearly `false` converts back and forth to `0`; it has to.) Would it be better to use `true` and `false`, or is it better to use `1` and `0`, so there's no confusion about the base type? On the other hand, using the number could lead to misinterpretation when reading a line like this that `inUse` is a count, rather than a flag.

As they say, the lower the stakes the greater the controversy.

Reply:
Agreed, using 1 and 0 might be confusing. Switching to 'true' and 'false' improves readability, so I'll stick with that for clarity.